### PR TITLE
Facility manager walkthrough

### PR DESCRIPTION
**Pull request by pwalczysko**
cc @jburel 

Similarly to the walkthrough for users, the landing page of the guides [under getting started](https://omero-guides.readthedocs.io/en/latest/#getting-started)should have a walkthrough for the facility managers.

This PR proposes such walkthrough.

The missing chapters will be added to the particular guides, such as omero-guide-introduction.

Edit:

I think we can go ahead and review.  Although this could certainly be improved, might do it in subsequent PRs.

### CODE REVIEW

**Comment by jburel** [comment]
let's be explicit in the file name: ``example_facilitymanager.rst``

**Comment by jburel** [comment]
@pwalczysko is this ready for review?

**Comment by pwalczysko** [comment]
> @pwalczysko is this ready for review?

I will relist tomorrow. My idea is to start adding features to the submodule repos, but in this process, I have to check that the whole construct still makes sense.

**Comment by jburel** [comment]
no rush. List only when ready

**Comment by pwalczysko** [comment]
Not sure how to get on here.
1. The last commit is one suggestion - basically cite the chapters inside long epics (like "Data publication" inside "Data management" epics) ?
2. Or split the long epics, creating new rst files, which would be "Data management for facility managers" and "Data publication" etc. ?

If I do Ad 2., how will that break or not break the redirects work from help.op..py.org ?

My worry is that especially "Data management" is so long that the facility manager will drown in it - no real difference in facility manager walkthrough to the users walkthrough - everybody has to read everything...

Maybe we should discuss. 

cc @jburel @will-moore

**Comment by jburel** [comment]
having more pages makes it easier to expose features from website for example.
The facility manager page is a collection of pages so we can redirect to faciliy-manager-walkthrough.html.
having more pages to describe data manage/data publication is happening in omero-guide-intro or other repo so that should not be a problem
Let's discuss on Monday

File: example_facility_manager.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
+OMERO walkthrough for facility managers
+=======================================
+
+In this document, we go through a common workflow that an imaging facility manager or other super-user of OMERO wishing to use OMERO and facilitate OMERO usage for others might follow:
```
[the remark this answers is not in the document]
Comment (pwalczysko) [reply example_facility_manager.rst]: fix pushed https://github.com/ome/omero-guides/pull/12/commits/c04620e79fc5457d44202d85868e5eca3046b367

**Comment by pwalczysko** [comment]
Fetched your changes and all looks good.

**Pull request closed without merging**